src/org/zaproxy/zap/extension/api/API.java

```diff
@@ -289,13 +289,12 @@ public boolean handleApiRequest (HttpRequestHeader requestHeader, HttpInputStrea
 
 					ApiResponse res;
 					switch (reqType) {
-					case action:	
-						// TODO Handle POST requests - need to read these in and then parse params from POST body
-						/*
-						if (getOptionsParamApi().isPostActions()) {
-							throw new ApiException(ApiException.Type.DISABLED);
+					case action:
+						if (requestHeader.getMethod().toUpperCase().equals("POST")) {
```

Comment:
It could use `HttpRequestHeader.POST` and it could use `equalsIgnoreCase` instead (in worst case it does not create a new String).